data/json/items/comestibles/carnivore.json

```diff
@@ -136,6 +136,31 @@
     "flags": [ "EATEN_HOT" ]
   },
   {
+    "id": "offal_pickled",
+    "copy-from": "offal",
+    "type": "COMESTIBLE",
+    "name": "pickled offal",
+    "name_plural": "pickled offal",
+    "description": "Cooked entrails, preserved in brine.  Packed with essential vitamins, tastes better than it smells.",
+    "parasites": 0,
+    "container" : "jar_glass",
+    "quench" : -1,
+    "fun": 0,
+    "stack_size" : 2
```

Comment:
It seems both recipes with output in jars are already set to use a `result_mult` of 2, does that mean the jars will already be filled to capacity?